plugins/azure/loadbalancer/lbHttpsOnly.js

```diff
@@ -15,13 +15,27 @@ module.exports = {
                 'App Service HTTPS redirection should be used to ensure site visitors ' +
                 'are always connecting over a secure channel.',
     },
+    settings: {
+        ignore_internal_loadbalancers: {
```

Comment:
```suggestion
        ignore_internal_lb_https: {
```
maybe we should add what we are checking for at the end instead of using multiple terms, name and description would stay the same. But attribute name will change.